src/state/moc/actions.js

```diff
@@ -0,0 +1,17 @@
+export const GET_MOCS = "GET_MOCS";
+export const GET_MOCS_SUCCESS = "GET_MOCS_SUCCESS";
+export const GET_MOCS_FAILED = "GET_MOCS_FAILED";
```

Comment:
nit, but could put these in a constants file instead of here.` moc/constants.js`.